core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2492,4 +2507,21 @@ private CarbonCommonConstants() {
    * property which defines the presto query default value
    */
   public static final String IS_QUERY_FROM_PRESTO_DEFAULT = "false";
+
+  /**
+   * the level 1 complex delimiter default value
+   */
+  public static final String COMPLEX_DELIMITERS_LEVEL_1_DEFAULT = "#";
```

Comment:
These default value and original value name I want change from DELIMITERS -> DELIMITER because string has delimiter (carbon.complex.delimiter.level.1)